event.ts

```diff
@@ -47,3 +47,24 @@ export async function prepareNormalNostrEvent<Kind extends NostrKind = NostrKind
     };
     return sender.signEvent<Kind>(event);
 }
+
+export async function prepareDeletionNostrEvent(
+    sender: NostrAccountContext,
+    ...events: NostrEvent<NostrKind>[]
+): Promise<NostrEvent<NostrKind.DELETE> | Error> {
+    if (events.some((e) => e.pubkey !== sender.publicKey.hex)) {
+        return new Error("deletion events must be from the same account");
+    }
+
+    const eTags = new Set(events.map((e) => e.id));
+
+    return prepareNormalNostrEvent(
+        sender,
+        {
+            kind: NostrKind.DELETE,
+            content: "",
```

Comment:
This is the reason of deletion for human readability, should pass in as an argument

Reply:
ok